packages/dashboard/src/components/resizablePanes/index.tsx

```diff
@@ -45,6 +56,10 @@ export const ResizablePanes: FC<ResizablePanesProps> = ({ leftPane, centerPane,
   /** Current widths of the three panes, in px */
   const [leftPaneWidth, setLeftPaneWidth] = useState(DEFAULT_SIDE_PANE_WIDTH);
   const [rightPaneWidth, setRightPaneWidth] = useState(DEFAULT_SIDE_PANE_WIDTH);
+  const selectedWidget = useSelector((state: DashboardState) => state.selectedWidgets);
+  const [RIGHT_PANE_COLLAPSED, setRightPaneCollapsed] = useState(selectedWidget?.length === 0);
+  const [LEFT_PANE_COLLAPSED, setLeftPaneCollapsed] = useState(selectedWidget?.length === 0);
```

Comment:
just a suggestion - a better name would be isLeftPaneCollapsed i think

Reply:
done